src/views/components/package-header.html

```diff
@@ -21,18 +21,18 @@ <h1 class="package-name">{{name}}</h1>
 				</a>
 				{{/if}}
 				{{#if githubRepo}}
-				<a target="_blank" rel="nofollow" href="https://github.com/{{githubRepo.user}}/{{githubRepo.project}}/" class="button" title="GitHub">
+				<a target="_blank" rel="nofollow" rel="noopener" href="https://github.com/{{githubRepo.user}}/{{githubRepo.project}}/" class="button" title="GitHub">
```

Comment:
This needs to be `rel="nofollow noopener"`

Reply:
Thanks, done